### ISSUE TITLE
Problem with the ruby parts in the install script

### ISSUE THREAD

**Issue by tmannen** [r888888888/danbooru#3012]
When trying to run the INSTALL.debian script on Ubuntu 16.04 I get errors:

```
...
* Installing rbenv...
bash: /root/.bash_profile: Permission denied
bash: rbenv: command not found
bash: /root/.bash_profile: Permission denied
bash: rbenv: command not found
* Generating secret keys...
unable to write 'random state'
unable to write 'random state'
* Installing gems...
bash: /root/.bash_profile: Permission denied
bash: gem: command not found
...
```

I'm completely new to ruby so not sure if this is some ruby specific issue. I assume this has something to do with the $PATH variable and how it works with sudo. I tried adding the installed rbenv bin path to the .bash_profile of root and running the script again, but it still fails with the same error messages. (The rest of the script seems to work fine but the ruby parts seem important).

**Comment by evazion** [comment]
It sounds like it's trying to install to `/root` when it should be installing to `/home/danbooru`. Try `grep danbooru /etc/passwd` to check that the home directory of the `danbooru` account is `/home/danbooru`. Also check `ls -lhd /home/danbooru` to make sure the permissions look good.

Installing rbenv isn't strictly necessary (that just installs a local copy of ruby into your home directory, but the ruby that comes with your distro will probably work), but the gems are necessary, those are Danbooru's ruby dependencies.

**Comment by tmannen** [comment]
Thanks for the reply. I ran those:

```
grep danbooru /etc/passwd
danbooru:x:1001:1001::/home/danbooru:/bin/bash

ls -lhd /home/danbooru
drwxr-xr-x 8 danbooru danbooru 4,0K touko  2 14:21 /home/danbooru
```

They seem correct.

I ended up doing it manually by using the created danbooru user. I ran the ruby related commands myself. Weird thing is that although .bash_profile should've added the rbenv created ruby to my path, it didn't do so, so I had to add it (might be Ubuntu related?).

Also, when I ran ```bundle install```, installing brypt didn't work right away. This was fixed by sudo apt-get install ruby-dev.

I needed to copy the database.yml.templ and danbooru_local_config.rb.templ to the config folder (removing the templ of course) before running bundle rake.

Also, I needed to run these as the danbooru user.

Now it works!
